### ISSUE TITLE
WordCount instead of Grains as a core exercise

### ISSUE THREAD

**Issue by F3PiX** [exercism/ruby#829]
Sorry for being late to this party. Triggered by the 'math discussion in  #4110 that excludes Grains. 
And related to revisiting the Ruby track (#823, #815)

I would be very happy to see 'math' topic added to Grains, make it `core: false` and make WordCount a core exercise instead.
Currently, the non-math solution to Grains is crucial in the curriculum to discuss Enumerable methods, while the Enumerables don't offer the optimal solution.  
WordCount is a better candidate for that in my opinion, because there are some suboptimal Enumerable options, that are easy to steer into the direction of the optimal Enumerables.

Apart from that, WordCount offers a lot more that should be introduced earlier in the track, like:
Initialize a class, extracting methods, public and private interface, attr_reader.
All of which get a quick follow up in Scrabble Score, that comes immediately afterwards.
So, any 'mentoring debt' can be taking care of there, and Scrabble also offers a more specialized Enumerable method, which builds easily on the skills acquired in WordCount. 

TL;DR How about marking Grains as a math topic and replace it with WordCount as core exercise?

**Comment by Insti** [comment]
Whatever you think is best. :smile:

**Comment by Insti** [comment]
@F3PiX where are we with this? Is this issue still relevant?

**Comment by F3PiX** [comment]
It is, but I'm currently working on a lot of more chances, so let's discuss them together. 
Thanks for the reminder :-)
